src/pages/home/base/mobile-core/lifecycle/behavior-reference.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Mobile Core Lifecycle behavior reference
+description: A reference for understanding how Lifecycle behaves under different scenarios.
+keywords:
+- Lifecycle
+- Lifecycle V1
+- Lifecycle for Mobile Core
+- Lifecycle for solution extensions
+- Lifecycle for Analytics
+- Lifecycle iOS
+- Lifecycle Android
+- Product reference
+---
+
+# Lifecycle behavior reference
+
+The Lifecycle extension relies solely on the Lifecycle events dispatched by the [`lifecycleStart`](api-reference.md#lifecyclestart) and [`lifecyclePause`](api-reference.md#lifecyclepause) calls to determine the start of new sessions and to calculate lifecycle data.
+
+## General behavior
+
+A Lifecycle session is the time between a `lifecycleStart` call and a `lifecyclePause` call. A new session is detected when `lifecycleStart` is called after a `lifecyclePause` and the [Lifecycle session timeout](configuration-keys.md#lifecyclesessiontimeout) has elapsed.
+
+All events that are not Lifecycle start or Lifecycle pause do not affect Lifecycle state or data. The starting of MobileCore via `registerExtensions` does not automatically start a Lifecycle session nor does it affect Lifecycle data.
+
+### Consecutive `lifecycleStart` API calls
+
+Consecutive `lifecycleStart` calls will not update the session start timestamp from the initial `lifecycleStart` call, and Lifecycle data is not changed.
+
+<InlineAlert variant="info" slots="text"/>
+
+In this case, a Lifecycle shared state is dispatched with its data reflecting the start timestamp from the initial `lifecycleStart` call. This can be verified in both application logs or Assurance.
+
+### Consecutive `lifecyclePause` API calls
+
+Consecutive `lifecyclePause` calls will update the persisted pause timestamp. On the next `lifecycleStart` API call, the last pause timestamp is used to determine if the previous session has timed out and a new session is started.
+
+## Expected Lifecycle scenarios
+
+The following Lifecycle workflows show examples of the expected ordering of Lifecycle events.
+
+**App first launch after installation**
+
+1. *App launched*
+2. `lifecycleStart` (Start of new session)
+3. `lifecyclePause`
+4. *App closed by user*
+
+![](./assets/android/lifecycle-first-launch.svg)
+
+<!-- mermaid.js diagram definition
+graph LR
+    A(["1.<br/>App launched"])
+    A ==> B("2.<br/><code>lifecycleStart</code><br>(Start of new session)")
+    B ==> C(3.<br/><code>lifecyclePause</code>)
+    C ==> D(["4.<br/>App closed by user"])
+
+    classDef dashedPill fill:#d3d3d3,stroke:#000,stroke-dasharray: 5 5,color:#000;
+    classDef regularPill fill:#3273de,stroke:#3273de,color:#fff;
+    classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
+    classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
+
+    class A,D regularPill;
+    class B,C regularBox;
+-->
+
+**Subsequent app launches, continue session with app close**
+
+1. Expected Lifecycle event(s)
```

Comment:
It's not clear to me what "Expected Lifecycle event(s)" is. If this is "App Launch" followed by "lifecycleStart" maybe state that/include in diagram instead?